pkg/controllers/logs/controller.go

```diff
@@ -36,9 +39,38 @@ func NewLogsController(log *zap.Logger, logsProvider logs.LogsProvider, router *
 	r.GET("", controller.Logs)
 	r.GET("/logs/namespace/:namespace/:entity/:entity_name", controller.FilterEntityLogs)
 	r.GET("/logs_by_labels/:labels", controller.FilterLabelLogs)
+	///r1 := router.Group("")
+	//r1.GET("/health", controller.HealthHandler)
+	//r1.GET("/ready", controller.ReadinessHandler)
 	return controller
 }
+func (controller *LogsController) HealthHandler(gctx *gin.Context) {
+	gctx.Header("Access-Control-Allow-Origin", "*")
+	gctx.Header("Access-Control-Allow-Headers", "access-control-allow-origin, access-control-allow-headers")
 
+	gctx.JSON(http.StatusOK, gin.H{"Message": "Success"})
+}
+
+func (controller *LogsController) ReadinessHandler(gctx *gin.Context) {
+	gctx.Header("Access-Control-Allow-Origin", "*")
+	gctx.Header("Access-Control-Allow-Headers", "access-control-allow-origin, access-control-allow-headers")
+	esAddress := flag.Lookup("es-addr").Value.(flag.Getter).Get().(string)
```

Comment:
I'd move this out of the controller. Controllers should receive an ES connection object, not create one inside. Just like it's done in the `LogsController`